pam/pam.go

```diff
@@ -1,24 +1,22 @@
-package main
+//go:generate go run github.com/msteinert/pam/cmd/pam-moduler -libname "pam_authd.so" -no-main
+//go:generate go generate --skip="pam_module.go"
```

Comment:
you missed me here?

Reply:
sooooo... The first command generates the `pam_module.go`, that includes other `go:generate` commands to actually build the library. Now if you run `go generate` again in this library it would of course be an infinite loop, so you need to skip the previously generated file.

In fact `go generate -C pam` just works as expected.

Indeed this would have been nice if the `pam_module.go` could go in another package, but being the shared library it's not possible.